api/tests/acceptance/application/competences/competences_test.js

```diff
@@ -400,4 +403,289 @@ describe('Acceptance | Route | competences', () => {
+          }],
+        })
+        .query({})
+        .matchHeader('authorization', 'Bearer airtableApiKeyValue')
+        .reply(200, { records: [airtableCompetence] });
+
+      generateNewId = vi.spyOn(idGenerator, 'generateNewId').mockReturnValueOnce('competence4');
+
+      const pixApiToken = 'secret';
+      nock('https://api.test.pix.fr')
+        .post('/api/token', { username: 'adminUser', password: '123', grant_type: 'password' })
+        .matchHeader('Content-Type', 'application/x-www-form-urlencoded')
+        .reply(200, { 'access_token': pixApiToken });
+      pixApiCacheScope = nock('https://api.test.pix.fr')
+        .patch('/api/cache/competences/competence4', {
+          id: 'competence4',
+          index: '2.2',
+          areaId: 'area2',
+          skillIds: [],
+          thematicIds: [],
+          origin: 'Pix',
+          name_i18n: {
+            fr: 'Quatrième compétence',
+            en: 'Fourth competence'
+          },
+          'description_i18n': {
+            fr: 'C’est la quatrième',
+            en: 'It’s the fourth one'
+          }
+        })
+        .matchHeader('Authorization', `Bearer ${pixApiToken}`)
+        .reply(200);
+    });
+
+    describe('when payload is NOT valid', () => {
+      it('should respond with status 400', async () => {
+        // given
+        const server = await createServer();
+
+        // when
+        const response = await server.inject({
+          method: 'POST',
+          url: '/api/competences',
+          payload:  {
+            data: {
+              type: 'competences',
+              attributes: {
+                title: 'Quatrième compétence',
+              },
+              relationships: {
+                area: {
+                  data: null,
+                },
+              },
+            },
+          },
+          headers: generateAuthorizationHeader(adminUser),
+        });
+
+        // then
+        expect(response.statusCode).toBe(400);
+      });
+    });
+
+    describe('when user is NOT admin', () => {
+      it('should respond with status 403', async () => {
+        // given
+        const server = await createServer();
+
+        // when
+        const response = await server.inject({
+          method: 'POST',
+          url: '/api/competences',
+          payload:  {
+            data: {
+              type: 'competences',
+              attributes: {
+                title: 'Quatrième compétence',
+              },
+              relationships: {
+                area: {
+                  data: {
+                    type: 'areas',
+                    id: 'recArea2',
+                  },
+                },
+              },
+            },
+          },
+          headers: generateAuthorizationHeader(editorUser),
+        });
+
+        // then
+        expect(response.statusCode).toBe(403);
+      });
+    });
+
+    describe('when area is unknown', () => {
+      it('should respond with status 400', async () => {
```

Comment:
`it('should respond with status 404',`

Reply:
Comme l’identifiant du domaine n’est pas dans l’URL, mais dans le body du POST, une 400 me semblait plus propre.

Reply:
Bien vu